base-theme/layouts/404.html

```diff
@@ -1,5 +1,20 @@
 {{ define "main" }}
-
-
-
+<div class="page-single" style="height: 100vh; display:flex; flex-direction:column">
```

Comment:
Sorry, I should have been more clear. We need stable CSS that doesn't require updating the 404 page content with each release. Otherwise, I don't care if it's inline, in the header, or something.